SConstruct

```diff
@@ -7,6 +7,7 @@ import sys
 import platform
 
 TICI = os.path.isfile('/TICI')
+Decider('make')
```

Comment:
can you try `MD5-timestamp`? It's safer and should still be much faster than just MD5